perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -260,12 +260,11 @@ sub has_permission_to {
     my ($self, $permission_type, $body_id) = @_;
 
     return 1 if $self->is_superuser;
+    return 0 unless $body_id;
```

Comment:
body_id should probably be renamed, as it's an array of one or more body IDs. (I think the `find` will still work, but should be checked.)